Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKTouchController.cs

```diff
@@ -0,0 +1,115 @@
+        };
+
+        /// <inheritdoc />
+        public override MixedRealityInteractionMapping[] DefaultRightHandedInteractions => new[]
+        {
+            new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer),
+            new MixedRealityInteractionMapping(1, "Axis1D.SecondaryIndexTrigger", AxisType.SingleAxis, DeviceInputType.Trigger),
+            new MixedRealityInteractionMapping(2, "Axis1D.SecondaryIndexTrigger Touch", AxisType.Digital, DeviceInputType.TriggerTouch),
+            new MixedRealityInteractionMapping(3, "Axis1D.SecondaryIndexTrigger Near Touch", AxisType.Digital, DeviceInputType.TriggerNearTouch),
+            new MixedRealityInteractionMapping(4, "Axis1D.SecondaryIndexTrigger Press", AxisType.Digital, DeviceInputType.TriggerPress),
+            new MixedRealityInteractionMapping(5, "Axis1D.SecondaryHandTrigger Press", AxisType.SingleAxis, DeviceInputType.TriggerPress),
+            new MixedRealityInteractionMapping(6, "Axis2D.SecondaryThumbstick", AxisType.DualAxis, DeviceInputType.ThumbStick),
+            new MixedRealityInteractionMapping(7, "Button.SecondaryThumbstick Touch", AxisType.Digital, DeviceInputType.ThumbStickTouch),
+            new MixedRealityInteractionMapping(8, "Button.SecondaryThumbstick Near Touch", AxisType.Digital, DeviceInputType.ThumbNearTouch),
+            new MixedRealityInteractionMapping(9, "Button.SecondaryThumbstick Press", AxisType.Digital, DeviceInputType.ThumbStickPress),
+            new MixedRealityInteractionMapping(10, "Button.One Press", AxisType.Digital, DeviceInputType.ButtonPress),
+            new MixedRealityInteractionMapping(11, "Button.Two Press", AxisType.Digital, DeviceInputType.ButtonPress),
+            new MixedRealityInteractionMapping(12, "Button.One Touch", AxisType.Digital, DeviceInputType.ButtonPress),
+            new MixedRealityInteractionMapping(13, "Button.Two Touch", AxisType.Digital, DeviceInputType.ButtonPress),
+            new MixedRealityInteractionMapping(14, "Touch.SecondaryThumbRest Touch", AxisType.Digital, DeviceInputType.ThumbTouch),
+            new MixedRealityInteractionMapping(15, "Touch.SecondaryThumbRest Near Touch", AxisType.Digital, DeviceInputType.ThumbNearTouch)
+        };
+
+
+        private static readonly ProfilerMarker UpdateButtonDataPerfMarker = new ProfilerMarker("[MRTK] OculusXRSDKController.UpdateButtonData");
+        protected override void UpdateButtonData(MixedRealityInteractionMapping interactionMapping, InputDevice inputDevice)
+        {
+            using (UpdateButtonDataPerfMarker.Auto())
+            {
+                base.UpdateButtonData(interactionMapping, inputDevice);
+
+                Debug.Assert(interactionMapping.AxisType == AxisType.Digital);
+
+                if (!(interactionMapping.InputType == DeviceInputType.TriggerTouch
```

Comment:
I'm not sure I fully understand what this `if` is doing. It reads out a `triggerData`, but it doesn't look like that's ever used. Can this be another case in the switch?

Reply:
It also looks like `DeviceInputType.TriggerTouch` gets handled in the base class already, though with a different `CommonUsages` feature

Reply:
fixed! though any idea why we're measuring float data for a digital axis type? (assuming is a WMR thing)
` if (interactionMapping.InputType == DeviceInputType.TriggerTouch
                    && inputDevice.TryGetFeatureValue(CommonUsages.trigger, out float triggerData))
{
    interactionMapping.BoolData = !Mathf.Approximately(triggerData, 0.0f);
}`

Reply:
Ah yep! Since the WMR motion controllers don't have capacitive touch sensors on the triggers, we instead report that it's "touched" as soon as the trigger is pressed to any amount. Since I don't see an equivalent `triggerTouch` in the [CommonUsages](https://docs.unity3d.com/ScriptReference/XR.CommonUsages.html) (though I was expecting one!), I'm not sure another way to do it.